_resources/port1.0/fetch/mirror_sites.tcl

```diff
@@ -303,82 +265,74 @@ set portfetch::mirror_sites::sites(freebsd) {
 # https://api.gentoo.org/mirrors/distfiles.xml appears to be valid now
 # Does not work: curl -s http://www.gentoo.org/main/en/mirrors2.xml | sed -n '/(http)\|(ftp)/s/.*"\([^"]*\)".*/    \1\/distfiles\/:nosubdir/p' | sed s@//distfiles@/distfiles@g
 set portfetch::mirror_sites::sites(gentoo) {
-    http://cosmos.illinois.edu/pub/gentoo/distfiles/:nosubdir
-    http://files.gentoo.gr/distfiles/:nosubdir
-    http://ftp-stud.hs-esslingen.de/pub/Mirrors/gentoo/distfiles/:nosubdir
-    http://ftp.daum.net/gentoo/distfiles/:nosubdir
-    http://ftp.dei.uc.pt/pub/linux/gentoo/distfiles/:nosubdir
-    http://ftp.fi.muni.cz/pub/linux/gentoo/distfiles/:nosubdir
-    http://ftp.halifax.rwth-aachen.de/gentoo/distfiles/:nosubdir
-    http://ftp.heanet.ie/pub/gentoo/distfiles/:nosubdir
-    http://ftp.iij.ad.jp/pub/linux/gentoo/distfiles/:nosubdir
-    http://ftp.jaist.ac.jp/pub/Linux/Gentoo/distfiles/:nosubdir
-    http://ftp.kaist.ac.kr/pub/gentoo/distfiles/:nosubdir
-    http://ftp.lanet.kr/pub/gentoo/distfiles/:nosubdir
-    http://ftp.linux.org.tr/gentoo/distfiles/:nosubdir
-    http://ftp.ntua.gr/pub/linux/gentoo/distfiles/:nosubdir
-    http://ftp.rnl.tecnico.ulisboa.pt/pub/gentoo/gentoo-distfiles/distfiles/:nosubdir
-    http://ftp.romnet.org/gentoo/distfiles/:nosubdir
-    http://ftp.snt.utwente.nl/pub/os/linux/gentoo/distfiles/:nosubdir
-    http://ftp.swin.edu.au/gentoo/distfiles/:nosubdir
-    http://ftp.twaren.net/Linux/Gentoo/distfiles/:nosubdir
-    http://ftp.uni-erlangen.de/pub/mirrors/gentoo/distfiles/:nosubdir
-    http://ftp.vectranet.pl/gentoo/distfiles/:nosubdir
-    http://gd.tuwien.ac.at/opsys/linux/gentoo/distfiles/:nosubdir
-    http://gentoo-euetib.upc.es/mirror/gentoo/distfiles/:nosubdir
-    http://gentoo.aditsu.net:8000/distfiles/:nosubdir
-    http://gentoo.bloodhost.ru/distfiles/:nosubdir
-    http://gentoo.c3sl.ufpr.br/distfiles/:nosubdir
-    http://gentoo.cs.uni.edu/distfiles/:nosubdir
+    http://gentoo.mirrors.tera-byte.com/distfiles/:nosubdir
+    https://mirror.csclub.uwaterloo.ca/gentoo-distfiles/distfiles/:nosubdir
     http://gentoo.gossamerhost.com/distfiles/:nosubdir
-    http://gentoo.inode.at/distfiles/:nosubdir
-    http://gentoo.iteam.net.ua/distfiles/:nosubdir
-    http://gentoo.mirror.web4u.cz/distfiles/:nosubdir
+    https://gentoo.osuosl.org/distfiles/:nosubdir
+    https://mirrors.evowise.com/gentoo/distfiles/:nosubdir
+    http://www.gtlib.gatech.edu/pub/gentoo/distfiles/:nosubdir
+    https://gentoo.ussg.indiana.edu/distfiles/:nosubdir
+    http://gentoo.mirrors.tds.net/gentoodistfiles/:nosubdir
+    http://gentoo.cs.utah.edu/distfiles/:nosubdir
     http://gentoo.mirrors.easynews.com/linux/gentoo/distfiles/:nosubdir
-    http://gentoo.mirrors.ovh.net/gentoo-distfiles/distfiles/:nosubdir
+    https://mirrors.rit.edu/gentoo/distfiles/:nosubdir
+    http://mirrors.rit.edu/gentoo/distfiles/:nosubdir
```

Comment:
There's no benefit to listing both the http and the https version of the same server. If the server supports both http and https (this one does), list the http one for the benefit of those users on old systems that cannot access modern https servers. If the server only supports https (if the http version redirects to the https version), list the https one.